operator/pkg/init.go

```diff
@@ -108,6 +108,13 @@ func NewInitJob(opt *InitOptions) *workflow.Job {
 			}
 		}
 
+		if opt.Components.Etcd.Local != nil && opt.Components.Etcd.Local.CommonSettings.Replicas != nil {
```

Comment:
Why replicas must be an odd number?

Reply:
We recommend the replicas of etcd as a odd number. 
@Poor12 Would it be better to change the error to warning. how do you think?